realm/realm-library/src/main/java/io/realm/RealmSchema.java

```diff
@@ -27,6 +27,8 @@
 import io.realm.internal.Util;
 import io.realm.internal.util.Pair;
 
+import static io.realm.RealmObjectSchema.SUPPORTED_SIMPLE_FIELDS;
```

Comment:
Please consider a normal import, and qualification at each use.  It makes the code so much clearer...